src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -209,6 +209,16 @@ public void onTraversalStart() {
             genomeLocSortedSet = new GenomeLocSortedSet(new GenomeLocParser(seqDictionary));
         }
         validationTypes = calculateValidationTypesToApply(excludeTypes);
```

Comment:
Also calculateValidationTypesToApply() should be changed to create a new list of the output valiation types it works on.